lib/middleware/room_middleware.dart

```diff
@@ -29,3 +29,34 @@ class CreateRoomAction extends MiddlewareAction {
         ordinals); // TODO: validate codes are unique for currently open rooms
   }
 }
+
+class EnterRoomAction extends MiddlewareAction {
+  @override
+  void handle(Store<GameModel> store, action, NextDispatcher next) {
+    loadGame(store);
+    navigatorKey.currentState
+        .push(new MaterialPageRoute(builder: (context) => new Game()));
```

Comment:
What does the new Game() do? Don't we want the game we've just loaded, when it loads?

Reply:
`Game` is just the name of the view that is an ongoing game. `loadGame()` is async so the constructor for `Game` shows a loading screen until the data has been pulled from firebase.